### ISSUE TITLE
Taurine Compatibility?

### ISSUE THREAD

**Issue by falsified** [iandwelker/smserver#128]
Switched from u0 to Taurine and seems to install but not run. A12, restored rootfs, removed all related files

**Comment by iandwelker** [comment]
SMServer should be completely compatible with Taurine, and I have had a few users confirm that it works for them on Taurine. However, I've also had a few people report crashes, so there may be an issue that only appears on Taurine that I haven't ran into yet. Would you be able to get me a crash report for this issue?

**Comment by falsified** [comment]
Hi Ian,

You know, this is where magic occurs sometimes and I am ok leaving it at just that. I've installed 0.8.0 from github and sileo multiple times, launched it and it would just crashout not providing any logs and the icon would disappear. I reinstalled cr4shed to try to get a log for you, and it required a reboot. After that reboot, it magically started working. I have uninstalled and rebooted, ldrestart, and resprung before and no luck, and magically after this attempt it worked.  So no idea what was happening  previous attempts! But thank you for your time in responding and giving support!
